hoomd/HOOMDMath.h

```diff
@@ -22,8 +22,33 @@
 #include <hoomd/extern/pybind/include/pybind11/pybind11.h>
 #endif
 
+#ifdef WIN32
```

Comment:
From earlier experience adding Windows compatibility for the GSD format, you must use `_WIN32` instead of `WIN32` for compatibility. CMake defines `WIN32` but the better way to detect is `_WIN32`. Let’s be consistent about that everywhere.
```suggestion
#ifdef _WIN32
```